src/main/java/com/wavefront/datastructures/Distribution.java

```diff
@@ -107,7 +107,7 @@ public static void normalizeCanonicalDistributions(@Nonnull List<Distribution> d
     if (Double.compare(percentsSum, HUNDRED_PERCENT) != 0) {
```

Comment:
Since we switched to double, please, also define HUNDRED_PERCENT as double.

Reply:
My investigation shows me:
1) Now (HUNDRED_PERCENT is int) we have cases when implicit conversion is done (int to double) and it make sense also define HUNDRED_PERCENT as double to prevent such conversions.
2)If we make HUNDRED_PERCENT as double then another cases will arise when we need explicit conversion(double to int)
In my opinion there is not sense make HUNDRED_PERCENT as double.

Reply:
My investigation shows me:

1) Now (HUNDRED_PERCENT is int) we have cases when implicit conversion is done (int to double) and it make sense also define HUNDRED_PERCENT as double to prevent such conversions.

2) If we make HUNDRED_PERCENT as double then another cases will arise when we need explicit conversion(double to int)
In my opinion there is not sense make HUNDRED_PERCENT as double.